app/views/shared/_why_host_toc.html.erb

```diff
@@ -0,0 +1,38 @@
+<section class="row text-center">
+  <h2>Why hosting a TOC</h2>
+  <p class="lead">Reasons to host</p>
+
+  <div class="col-md-4 text-left">
+    <div class="row">
+      <div class="col-md-4">
+        <%= image_tag '', class: "img-small img-circle" %>
+      </div>
+      <div class="col-md-8">
+        <h4>Text title</h4>
+        Description text
```

Comment:
I think if these descriptions were in a `<p>` or a `<span>`, we would have more flexibility for styling (if we needed it).